src/components/app/data/services/enterpriseCustomerUser.js

```diff
@@ -204,3 +204,16 @@ export async function updateUserCsodParams({ data }) {
   const url = `${getConfig().LMS_BASE_URL}/integrated_channels/api/v1/cornerstone/save-learner-information`;
   return getAuthenticatedHttpClient().post(url, data);
 }
+
+/**
+ * Helper function to uunlink enterprise customer user by making a POST API request.
```

Comment:
The doc string seems to have a couple of inconsistencies and could be improved for clarity...
Suggestion:
```
/**
 * Helper function to unlink an enterprise customer user by making a POST API request.
 * @param {string} enterpriseCustomerUserUUID - The UUID of the enterprise customer user to be unlinked.
 * @param {string} userEmail - The email of the user that should be unlinked.
 * @returns {Promise} - A promise that resolves when the user is successfully unlinked from the enterprise customer.
 * @throws {Error} - Throws an error if the unlinking process fails.
 */
```